src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -337,6 +337,41 @@ class WelcomePage extends Disposable {
 		this.createDropDown();
 		this.createWidePreviewToolTip();
 		this.createPreviewModal();
+		this.handleAccessibility(this.commandService);
```

Comment:
you probably don't need to pass the command service to the method  since it is already a private property of the class